Parsers/AADUserInfo/AADUserInfo.txt

```diff
@@ -0,0 +1,40 @@
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as AADUserInfo.
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. AADUserInfo | take 10).
+// Reference : Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+AADUserInfo_CL
```

Comment:
Sample data not provided. Getting error ("'summarize' operator: Failed to resolve table or column expression named 'AADUserInfo_CL'") in parser deployment in log analytics workspace. 

Please fix the issue.

Reply:
added CL schema